### PR TITLE
Jenkins Build Process

### PR DESCRIPTION
**Pull request by osalpekar**

### CODE REVIEW

**Comment by devin-petersohn** [comment]
Jenkins, add to whitelist.

**Comment by simon-mo** [comment]
@osalpekar can we not install miniconda to /home/jenkins? our sysadmins will be very sad with we are messing the build environment this way. I can write up dockerfile for you so we run tests in docker containers.

**Comment by simon-mo** [comment]
jenkins test this please

**Comment by simon-mo** [comment]
jenkins test this please

**Comment by simon-mo** [comment]
jenkins test this please

**Comment by simon-mo** [comment]
jenkins test this please

**Comment by shaneknapp** [comment]
test this please

**Comment by shaneknapp** [comment]
this LGTM, but i'd like to have a conversation outside of this issue regarding your build strategies...  i'd really like to see a minimum of two builds for modin:  one for pull requests like this, and one for `master`.  please feel free to swing by my desk @ RISE and we can spend a few minutes chatting about this.

thanks for dockerizing the build, btw.  the future is now!  :)

File: .jenkins/run_test.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+set -x
+
+pip install awscli pytest-html
+python .jenkins/inject_aws_credentials.py
```
Comment (shaneknapp) [inline .jenkins/run_test.sh]: i'm not entirely sure that you need to do this...  the AWS login creds are passed in through env vars and should be available to the aws-cli commands later in the script.

**Pull request merged**